quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/GetCollectionsApiController.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.diging.quadriga.api.v1;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import edu.asu.diging.quadriga.aspect.annotation.InjectToken;
+import edu.asu.diging.quadriga.config.web.TokenInfo;
+import edu.asu.diging.quadriga.core.model.Collection;
+import edu.asu.diging.quadriga.core.service.CollectionManager;
+
+@Controller
+public class GetCollectionsApiController {
```

Comment:
Let's call this `CollectionsApiControlller` and if we need other CRUD methods we can put them in here.